content/SCALETutorials/Instances/_index.md

```diff
@@ -19,554 +19,657 @@ keywords:
 
-### Adding and Removing Devices
-After creating the VM, you can add or remove virtual devices.
+- [Create one or more zvols]({{< relref "/SCALETutorials/Datasets/AddManageZvols.md" >}}) on a storage pool for the virtual disk now or do this from the **Volumes** screen while configuring the instance.
 
-Click on the VM row on the **Virtual Machines** screen to expand it and show the options, then click <i class="material-icons" aria-hidden="true" title="Devices">device_hub</i> **Devices**.
+- Compare the recommended specifications for the guest operating system with your available host system resources.
+  Reference these when allocating resources to the instance.
 
-{{< trueimage src="/images/SCALE/Virtualization/VMDevicesListed.png" alt="Devices" id="Devices" >}}
+{{< /hint >}}
 
-{{<include file="/static/includes/addcolumnorganizer.md">}}
+To create a new virtual machine, from the **Create Instance** screen:
 
-Device notes:
+1. Configure the instance configuration settings.
 
-* A virtual machine attempts to boot from devices according to the **Device Order**, starting with **1000**, then ascending.
-* A **CD-ROM** device allows booting a VM from a CD-ROM image like an installation CD.
-  The CD image must be available in the system storage.
+   {{< trueimage src="/images/SCALE/Virtualization/InstanceConfigurationVM.png" alt="Instance Configuration - VM" id="Instance Configuration - VM" >}}
 
-<!-- nb: this is the content from the removed /scaletutorials/virtualization/addmanagevmdevicesscale/ -->
-<!-- preserved here in case any of it is useful when this tutorial is rewritten for Instances. -->
-<!--
-## Managing Devices
+   a. Enter a name for the virtual machine.
 
-After creating a VM, the next step is to add virtual devices for that VM.
-Using the Create Virtual Machine wizard configures at least one disk, NIC, and the display as part of the process.
-To add devices, from the **Virtual Machines** screen, click anywhere on a VM entry to expand it and show the options for the VM.
+      {{< include file="/static/includes/InstanceNameRequirements.md" >}}
 
-Click <i class="material-icons" aria-hidden="true" title="Devices">device_hub</i> **Devices** to open the **Devices** screen for the VM.
-From this screen, you can edit, add, or delete devices.
-Click the <span class="material-icons">more_vert</span> icon at the right of each listed device to see device options.
+   b. Select **VM** as the **Virtualization Method**.
 
-{{< trueimage src="/images/SCALE/Virtualization/VirtualMachinesScreenwithVMDetails.png" alt="VM Options" id="VM Options" >}}
+   c. Select one of the available **VM Image Options**: **Use a Linux Image** or **Upload ISO, import a zvol or use another volume**.
 
-The devices for the VM display as a list.
+      - Selecting **Use a Linux Image**:
 
-Device notes:
+         Select this option to create the VM using a Linux image from [linuxcontainers.org](https://linuxcontainers.org/).
```

Comment:
This looks odd when rendered. I suggest combining these two lines into one bullet point, like this as one way to do it:
(it is not letting me use the option to add it so I'll just type it out)

   - Select **Use a Linux Image** to create the vM using a Linux image from [linuxcontainers.org](https://linuxcontainers.org/).